.circleci/config.yml

```diff
@@ -39,6 +39,10 @@ jobs:
       BUCKET: << parameters.bucket >>
     steps:
       - checkout
+      - restore_cache:
+          keys:
+            - pach-build-dependencies-v2-{{ checksum "etc/testing/circle/install.sh" }}
+            - pach-build-dependencies-v2-
```

Comment:
The `pach-build-dependencies-v2-` is the problem here, we got rid of this in master.